doc/openapi-user-guide.md

```diff
@@ -149,22 +149,43 @@ spec:
-| serviceAccountsEnabled    | no           | for oidc only                                                                                                                                                                                                                   |
-| directAccessGrantsEnabled | no           | for oidc only                                                                                                                                                                                                                   |
+| **Field**                | **Required** | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+|---------------------------|--------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| issuerType               | no           | Valid values: [keycloak, rest]. Defaults to `rest`                                                                                                                                                                                                                                                                                                                                                                                                    |
+| issuerEndpoint           | no           | issuerEndpoint can be defined in `issuerEndpointRef` or as plain value (please see CR example and notes below). The format of this endpoint is determined on your OpenID Provider setup. For RHSSO:  https://<client_id>:<client_secret>@<host>:<port>/auth/realms/<realm_name>                                                                                                                                                                       |
+| issuerEndpointRef        | no           | The secret that contains  `issuerEndpoint`                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
explain that `issuerEndpoint` takes precedence over `issuerEndpointRef`

Reply:
it explained few lines below

Reply:
@eguzki could you please suggest what to write here - 
? maybe issuerEndpoint: https://3scale-zync:<secret-name>@keycloak-rhsso-test.example.com/auth/realms/petstore